arangod/IResearch/ViewSnapshot.cpp

```diff
@@ -176,7 +171,7 @@ ViewSnapshot* makeViewSnapshot(transaction::Methods& trx, void const* key,
 
   for (auto const& link : links) {
     if (!link) {
-      LOG_TOPIC("b054e", WARN, TOPIC)
+      LOG_TOPIC("fffff", WARN, TOPIC)
```

Comment:
Shouldn't the ID remain the same?

Reply:
Nope

Reply:
This log message was copied by Jan (from compute), so now it cannot triggered in compute.

Jan should moved old logging (from compute to here).

I fix this.

But in general it's not important both check was added in 3.10

(I just like ffff more)